meta/cliffparnitzky/tiny-mce-bundle-all/en.yml

```diff
@@ -0,0 +1,10 @@
+en:
+    title: Tiny MCE Bundle all
```

Comment:
Do you want to intentionally write it that way (i.e "Tiny MCE" instead of "TinyMCE")?

Reply:
I will correct it